xamarin/Xamarin.Forms
# [WPF] Use project style SDK; target netcoreapp3.0

This adds .netcoreapp3.0 besides the existing net 4.6.1 framework target for the WPF backend.

### Description of Change ###

Add target framework .netcoreapp3.0.
Replaces the old Platform.WPF project file  with an project sdk style one.
Update the OpenTK.GLControl package to 3.0.1.

### Issues Resolved ### 

fixes https://github.com/xamarin/Xamarin.Forms/issues/5092

### API Changes ###
None 

### Platforms Affected ### 

WPF

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

1. Build the project
2. Test the library with WPF/.NET Core 3
  * Create a WPF/.NET Core 3 app and reference the created dll's
  * Add the logic to init XF and show a XF ContentPage
  * Run the project -> The content page should appear

3. Test the library with WPF/.NET Framework
  * Create a WPF/.NET Framework app and reference the created dll's
  * Add the logic to init XF and show a XF ContentPage
  * Run the project -> The content page should appear

### PR Checklist ###
